.github/workflows/nightly.yml

```diff
@@ -15,6 +15,55 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install ${{ matrix.rust }}
         uses: actions-rs/toolchain@v1
+      - name: Start minikube
```

Comment:
minikube is not needed.  assume CI has run smoke test for all check-in.
just need to bring musl

Reply:
0817739 removes the extraneous steps; only keeps musl install step;